test/Microsoft.AspNet.Mvc.ModelBinding.Test/Validation/DataAnnotationsModelValidatorProviderTest.cs

```diff
@@ -172,41 +178,15 @@ public void IValidatableObjectGetsAValidator()
             var mockValidatable = new Mock<IValidatableObject>();
             var metadata = _metadataProvider.GetMetadataForType(mockValidatable.Object.GetType());
 
-            // Act
-            var validators = provider.GetValidators(metadata);
-
-            // Assert
-            Assert.Single(validators);
-        }
-#endif
-
-        // Integration with metadata system
-
-        [Fact]
-        public void DoesNotReadPropertyValue()
```

Comment:
Was this test not interesting? Looks like this ensured that we do not inspect the value while fetching the validator?